x/rollapp/client/cli/tx_create_rollapp.go

```diff
@@ -66,7 +66,9 @@ func CmdCreateRollapp() *cobra.Command {
 				metadatas,
 				genesisAccounts,
 			)
-
+			if err := msg.ValidateBasic(); err != nil {
```

Comment:
I believe this is already done automatically as part of the `GenerateOrBroadcastTxCLI` so no need to call it.